packages/retail-ui-extensions/src/extension-api/cart-api/cart-api.ts

```diff
@@ -82,7 +82,7 @@ export interface CartApiContent {
    * @param amount the percentage or fixed monetary amount deducted with the discout
    */
   setLineItemDiscount(
-    uuid: number,
+    uuid: string,
     type: DiscountType,
```

Comment:
Should we also address the DiscountType while we're here? This type includes DiscountCode, which is not valid for line item discounts. There should be a separate type that excludes DiscountCode to account for this.

Reply:
good point